lib/project_types/script/commands/create.rb

```diff
@@ -9,6 +9,7 @@ class Create < ShopifyCli::SubCommand
         parser.on("--extension_point=EP_NAME") { |ep_name| flags[:extension_point] = ep_name }
         parser.on("--extension-point=EP_NAME") { |ep_name| flags[:extension_point] = ep_name }
         parser.on("--language=LANGUAGE") { |language| flags[:language] = language }
+        parser.on("--no-config-ui") { |no_config_ui| flags[:no_config_ui] = no_config_ui }
```

Comment:
Nit: Is there a reason why the flag is named `no_config_ui` instead of `no_ui_config` the latter seems more intuitive. I might be missing something though.

Reply:
Naming it `ui-config` is in sync with the file name too.

Reply:
I find the naming of the concept a bit confusing as well. I _think_ I prefer `config_ui` since we are defining the traits of the UI that allows us to update the config. In the backend we also store it as a `config_ui_definition`. 

Though, I'm not attached to the name if you still prefer `ui_config`?

> Naming it ui-config is in sync with the file name too.

You're right, we should be consistent with the filename either way.

Reply:
Talked on slack, we will use `config-ui.yml` as the filename for consistency and leave everything else as `config_ui`.